### DISCUSSION TITLE
Older Firmware 2022 or 2023 for 427

### DISCUSSION THREAD

**Discussion by geekazine** [mriscoc/Ender3V2S1#1387] [Q&A]
I tried installing the 2024 427 BLT UBL firmware on my 3V2 and it bricked the machine. I had to put the Creality Base 427 on to get it back to working. 

The printer had a 2022 version of the firmware, but I don't have a copy of that. As far as I can tell, older firmware is removed. 

Does anyone have the 427 BLT UBL firmware from 2022 or maybe 2023? Or am I missing something?

**Comment by mriscoc** [comment]
Verify if the problem is the printer board or only the screen, connect the printer to a terminal (like Pronterface), check: https://github.com/mriscoc/Ender3V2S1/issues/966

**Comment by geekazine** [comment]
You're asking me to go down rabbit holes I don't normally go down. Spending hours debugging is just not something I have time for. 

That is why I was hoping to get the 427 BLT from 2022, when it worked. Get back to printing, then figure out the problem later. 

I can tell you Octoprint doesn't recognize a printer when I try to install the latest firmware. When I go back to the Creality stock, it sees the printer fine.
